pandas/core/accessor.py

```diff
@@ -1,8 +1,6 @@
 """
-
-accessor.py contains base classes for implementing accessor properties
+Accessor.py contains base classes for implementing accessor properties
```

Comment:
I don't think we're automatically validating module level docstrings, you can restore the actual name of the file `accessor.py` and not the capitalized version of it.

Reply:
Fixed.